api/vulnerability_exceptions.go

```diff
@@ -446,11 +446,17 @@ type VulnerabilityExceptionPackage struct {
 	Version string
 }
 
-func (vc VulnerabilityExceptionCriteria) FixableEnabled() bool {
+func (vc VulnerabilityExceptionCriteria) FixableEnabled() *bool {
+	if vc.Fixable == nil {
+		return nil
+	}
+
 	if len(vc.Fixable) > 0 {
-		return vc.Fixable[0] == 1
+		truePtr := vc.Fixable[0] == 1
+		return &truePtr
 	}
-	return false
+	falsePtr := false
+	return &falsePtr
```

Comment:
Can we write a few unit tests to verify the three things/options we want this function to return.

1) When it is false, return false
1) When it is true, return true
1) When it is not present, return nil
1) When it is empty, return nil (but I don't know if this is possible or, what if the APIs return empty when we set it and then remove it)

Reply:
Maybe to remove the `fixable == true` or `fixable == false` we need to use a `PUT`